common/source/docs/common-redundant-telemetry.rst

```diff
@@ -0,0 +1,56 @@
+.. _common-redundant-telemetry:
+
+=========================
+Redundant Telemetry Links
+=========================
+
+Many high value vehicles utilize a single telemetry link for vehicle RC control, telemetry, and, often, video. While a properly setup GCS failsafe, will often return a vehicle safely, it may not allow a safe landing if that link remains non-operational upon the return.
+
+Therefore, having redundant telemetry links, is often a wise investment. For example, if the primary link is LTE based, a long range 900 Mhz or 433Mhz additional link may be a worthwhile investment. Even a shorter range WIFI link would allow re-establishing control upon return to home from a properly setup GCS failsafe in order to allow control for a safe landing.
```

Comment:
```suggestion
Therefore, having redundant telemetry links is often a wise investment. For example, if the primary link is LTE based, an additional long range 900 Mhz or 433Mhz link may be a worthwhile investment. Even a shorter range WIFI link would allow re-establishing control upon return to home from a properly setup GCS failsafe in order to allow for a safe, controlled landing.
```